stablehlo/reference/Element.cpp

```diff
@@ -200,42 +171,59 @@ Element Element::operator+(const Element &other) const {
       });
 }
 
-Element ceil(const Element &e) {
-  APFloat val = getFloatValue(e);
+Element ceil(const Element &el) {
+  APFloat val = el.getFloatValue();
   val.roundToIntegral(APFloat::rmTowardPositive);
-  return Element(e.getType(), FloatAttr::get(e.getType(), val));
+  return Element(el.getType(), val);
 }
 
-Element floor(const Element &e) {
-  APFloat val = getFloatValue(e);
+Element floor(const Element &el) {
+  APFloat val = el.getFloatValue();
   val.roundToIntegral(APFloat::rmTowardNegative);
-  return Element(e.getType(), FloatAttr::get(e.getType(), val));
+  return Element(el.getType(), val);
 }
 
-Element cosine(const Element &e) {
+Element cosine(const Element &el) {
   return mapWithUpcastToDouble(
-      e, [](double e) { return std::cos(e); },
+      el, [](double e) { return std::cos(e); },
       [](std::complex<double> e) { return std::cos(e); });
 }
 
-Element sine(const Element &e) {
+Element sine(const Element &el) {
   return mapWithUpcastToDouble(
-      e, [](double e) { return std::sin(e); },
+      el, [](double e) { return std::sin(e); },
       [](std::complex<double> e) { return std::sin(e); });
 }
 
-Element tanh(const Element &e) {
+Element tanh(const Element &el) {
   return mapWithUpcastToDouble(
-      e, [](double e) { return std::tanh(e); },
+      el, [](double e) { return std::tanh(e); },
       [](std::complex<double> e) { return std::tanh(e); });
 }
 
-void Element::print(raw_ostream &os) const { value_.print(os); }
+void Element::print(raw_ostream &os) const {
+  if (isSupportedIntegerType(type_)) {
+    IntegerAttr::get(type_, getIntegerValue()).print(os);
```

Comment:
To be honest, I think we should print attributes with `/*elideType=*/ true` to make things easier on the eyes, but I can make this change in a separate PR when this PR lands.

Reply:
Thanks!